config/webpacker.yml

```diff
@@ -3,26 +3,47 @@
 default: &default
   source_path: app/javascript
   source_entry_path: packs
+  public_root_path: public
```

Comment:
Part of the webpacker upgrade is to merge the current webpacker.yml file with [this one](https://github.com/rails/webpacker/blob/master/lib/install/config/webpacker.yml).